sizefashion/src/reducers/filter_reducer.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  LOAD_PRODUCTS,
+  UPDATE_FILTERS,
+  FILTER_PRODUCTS,
+  CLEAR_FILTERS,
+} from '../actions'
+import { initialStateType } from '../context/filter_context'
+
+const filter_reducer = ( state: initialStateType, action: { type: any; payload?: any } ) => {
```

Comment:
@otaaviostein avoid calling your functions using snake_case. Prefer camelCase instead. Same goes for the file structure!